python3/36.py

```diff
@@ -0,0 +1,53 @@
+class Solution:
```

Comment:
**PaiR(PR AI Reviewer) :** `List` is not defined. You need to import it from the `typing` module.